tests/lambda_endpoints/conftest.py

```diff
@@ -27,6 +27,7 @@ def _set_env(monkeypatch):
     # Mock the api gateway url
     # This is used in batch_starter.py
     monkeypatch.setenv("IMAP_DATA_ACCESS_URL", "https://test.url")
+    monkeypatch.setenv("EFS_SPICE_MOUNT_PATH", "/tmp")
```

Comment:
This should be a temporary directory in tests (bring in the fixture above).
```suggestion
    monkeypatch.setenv("EFS_SPICE_MOUNT_PATH", str(tmpdir))
```